source/fuzz/fuzzer_util.h

```diff
@@ -151,7 +151,12 @@ uint32_t GetBoundForCompositeIndex(const opt::Instruction& composite_type_inst,
 
 // Returns true if and only if |context| is valid, according to the validator
 // instantiated with |validator_options|.
-bool IsValid(opt::IRContext* context, spv_validator_options validator_options);
+bool IsValid(const opt::IRContext* context,
+             spv_validator_options validator_options);
+
+bool IsValidAndWellFormed(const opt::IRContext* context,
```

Comment:
Missing comments.

Reply:
Fixed.

Reply:
I also decided to make IsValid take a message consumer, which led to another pretty big change propagating through a lot of files.